oms/azuredeploy.json

```diff
@@ -0,0 +1,1210 @@
+              "recoveryPointHistory": 1440,
+              "appConsistentFrequencyInMinutes": 60,
+              "multiVmSyncStatus": "Enable"
+            }
+          }
+        },
+        {
+          "apiVersion": "2016-06-01",
+          "name": "DefaultVMwarePolicy-failback",
+          "type": "replicationPolicies",
+          "id": "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', resourceGroup().name, '/providers/Microsoft.RecoveryServices/vaults/', parameters('omsRecoveryVaultName'), '/replicationPolicies/', 'DefaultVMwarePolicy-failback')]",
+          "dependsOn": [
+            "[concat('Microsoft.RecoveryServices/vaults/', parameters('omsRecoveryVaultName'))]",
+            "[concat('Microsoft.RecoveryServices/vaults/', parameters('omsRecoveryVaultName'), '/replicationPolicies/DefaultVMwarePolicy')]"
+          ],
+          "properties": {
+            "subscriptionid": "[subscription().subscriptionId]",
+            "friendlyName": "DefaultVMwarePolicy-failback",
+            "providerSpecificInput": {
+              "instanceType": "InMage",
+              "recoveryPointThresholdInMinutes": 15,
+              "recoveryPointHistory": 1440,
+              "appConsistentFrequencyInMinutes": 60,
+              "multiVmSyncStatus": "Enable"
+            }
+          }
+        },
+        {
+          "apiVersion": "2016-06-01",
+          "name": "DefaultHyperVPolicy",
+          "type": "replicationPolicies",
+          "id": "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', resourceGroup().name, '/providers/Microsoft.RecoveryServices/vaults/', parameters('omsRecoveryVaultName'), '/replicationPolicies/', 'DefaultHyperVPolicy')]",
+          "dependsOn": [
+            "[concat('Microsoft.RecoveryServices/vaults/', parameters('omsRecoveryVaultName'))]"
+          ],
+          "properties": {
+            "subscriptionId": "[subscription().subscriptionId]",
+            "friendlyName": "DefaultHyperVPolicy",
+            "providerSpecificInput": {
+              "instanceType": "HyperVReplicaAzure",
+              "recoveryPointHistoryDurationInHours": 2,
+              "applicationConsistentSnapshotFrequencyInHours": 0,
+              "replicationInterval": 900,
+              "onlineReplicationStartTime": "",
+              "activeStorageAccountId": "[reference(resourceId('Microsoft.Storage/storageAccounts/',variables('recoveryStorageAccountName')))]",
+              "encryption": "Disable"
+            }
+          }
+        }
+      ]
+    },
+    {
+      "apiVersion": "2015-06-15",
+      "location": "[parameters('omsRecoveryVaultRegion')]",
+      "name": "[variables('recoveryStorageAccountName')]",
+      "type": "Microsoft.Storage/storageAccounts",
+      "comments": "Storage account for ASR",
+      "properties": {
+        "accountType": "[variables('recoveryStorageType')]"
+      }
+    },
+    {
+      "apiVersion": "2015-11-01-preview",
```

Comment:
not at this time. this is to showcase some new capabilities which is only present in these preview versions.